extensions/wikia/WikiaApi/WikiaApiAjaxLogin.php

```diff
@@ -96,6 +96,10 @@ public function execute() {
 					$result['result'] = 'Throttled';
 					$result['text'] = wfMsg('login-throttled');
 					break;
+				case LoginForm :: ABORTED :
+					$result['result'] = 'Aborted';
+					$result['text'] = wfMsg($loginForm->mAbortLoginErrorMsg);
```

Comment:
In this file (eg. 3 lines above) wfMsg() is used in all places so I'd keep it for the file-level consistency. Sound reasonable?